### ISSUE TITLE
Ubuntu Mono font looks the wrong size

### ISSUE THREAD

**Issue by memeplex** [microsoft/vscode#88916]
- VSCode Version: 1.41.1
- OS Version: Ubuntu 19.04

Steps to Reproduce:

1. Set "Ubuntu Mono" as font
2. With the default size (14) it should look small (at least in a hidpi screen with a 2x scaling factor)
3. Increase its size to 16
4. Now parts of the GUI show way too big fonts (documentation popups, python interactive window output, etc)

Does this issue occur when all extensions are disabled?: Yes

So, to recap: I'm unable to use Ubuntu Mono: it's either too small or it plays bad with other parts of the GUI. My screen is 3000x2000.

**Comment by rebornix** [comment]
@memeplex can you help a bit more on understanding how changing editor font size change the size the of other parts of VS Code?
